lib/ember-cli/engine.rb

```diff
@@ -6,7 +6,7 @@ class Engine < Rails::Engine
 
     initializer "ember-cli-rails.inflector" do
       ActiveSupport::Inflector.inflections :en do |inflect|
```

Comment:
@rwz actually, `.inflections` is a problem as well, since i *doesn't* take arguments in 3.1